roles/common/defaults/main.yml

```diff
@@ -214,86 +242,88 @@ gi_interim_patches:
-  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.7.0.0.200414", patchnum: "30783556", patchfile: "p30783556_190000_Linux-x86-64.zip", patch_subdir: "/30805684", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE }
-  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.8.0.0.200714", patchnum: "31326369", patchfile: "p31326369_190000_Linux-x86-64.zip", patch_subdir: "/31219897", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE }
-  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.9.0.0.201020", patchnum: "31720429", patchfile: "p31720429_190000_Linux-x86-64.zip", patch_subdir: "/31668882", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE }
+  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.3.1.0.0", patchnum: "29859737", patchfile: "p29859737_190000_Linux-x86-64.zip", patch_subdir: "/29799057", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: FALSE, md5sum: "OwF/UXNB31s16fvZDx9Jqg==" }
+  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.3.1.0.0", patchnum: "29859737", patchfile: "p29859737_190000_Linux-x86-64.zip", patch_subdir: "/29548437", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE, md5sum: "OwF/UXNB31s16fvZDx9Jqg==" }
+  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.4.0.0.190716", patchnum: "29699079", patchfile: "p29699079_190000_Linux-x86-64.zip", patch_subdir: "/29774421", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE, md5sum: "QM2iujCI5hpsEvEWdAuRRQ==" }
+  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.5.0.0.191015", patchnum: "30133178", patchfile: "p30133178_190000_Linux-x86-64.zip", patch_subdir: "/30128191", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE, md5sum: "QYnK6uhQp8QZH90/pMCvag==" }
+  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.6.0.0.200114", patchnum: "30463609", patchfile: "p30463609_190000_Linux-x86-64.zip", patch_subdir: "/30484981", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE, md5sum: "Cy964W9iPo0mkFrnumALBg==" }
+  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.7.0.0.200414", patchnum: "30783556", patchfile: "p30783556_190000_Linux-x86-64.zip", patch_subdir: "/30805684", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE, md5sum: "9PajEXNsYfGqI7GU+XscqQ==" }
+  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.8.0.0.200714", patchnum: "31326369", patchfile: "p31326369_190000_Linux-x86-64.zip", patch_subdir: "/31219897", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE, md5sum: "6Vm4aGIszIQ7NzbuNDBFEw==" }
+  - { category: "RU_Combo", base: "19.3.0.0.0", release: "19.9.0.0.201020", patchnum: "31720429", patchfile: "p31720429_190000_Linux-x86-64.zip", patch_subdir: "/31668882", prereq_check: TRUE, method: "opatch apply", ocm: FALSE, upgrade: TRUE, md5sum: "tTZDYSasdnt7lrNJ/MYm1g==" }
```

Comment:
Can a line about calculating the new md5 sum be added in the recently merged file docs/patch-metadata.md?

Reply:
Done, thanks!